cmd/ext/kgflags.h

```diff
@@ -373,15 +373,15 @@ bool kgflags_parse(int argc, char **argv) {
         _kgflags_parse_flag(flag, prefix_no);
     }
 
-    _kgflags_assign_default_values();
```

Comment:
Not sure about these changes either, this is in a third-party repo, should probably start by updating that to latest.
Also, by moving _kgflags_assign_default_values the behavior is changed I guess?
Flags that are required but have a default value can't fall back to default now, right? You *must* provide a value even if it has default.
Debatable weather a flag should be able to be set both required and have a default, but the behavior of is changed with this, right?

Reply:
Yeah, I'm not sure what the deal is here.... but for me defaults didn't work at all.. So I just tried a few things until it works "for me"... I think in general this library is just really bad... there is better out there, or even just doing a custom roll would be easier to understand.